oxd-server/src/test/java/org/gluu/oxd/server/RegisterSiteTest.java

```diff
@@ -95,4 +95,27 @@ public static RegisterSiteResponse registerSite(ClientInterface client, String o
         assertTrue(!Strings.isNullOrEmpty(resp.getOxdId()));
         return resp;
     }
+
+    public static RegisterSiteResponse registerWithMultipleRedirectUrls(ClientInterface client, String opHost, String authorizationRedirectUrl, String postLogoutRedirectUrl, String logoutUrl, String redirectUrls) throws IOException {
```

Comment:
we don't need this copy/paste. Simply change existing `registerSite` to accept `List<String> redirectUris` and then in code that invokes it parse and pass list `Lists.newArrayList(redirectUrls.split(","))`.

Reply:
Changes done.